packages/flutter/test/widgets/slivers_padding_test.dart

```diff
@@ -18,20 +18,20 @@ class _MockRenderSliver extends RenderSliver {
 
 }
 
-Future<void> test(WidgetTester tester, double offset, EdgeInsetsGeometry padding, AxisDirection axisDirection, TextDirection textDirection) {
+Future<void> test(WidgetTester tester, double offset, EdgeInsetsGeometry padding, AxisDirection axisDirection, TextDirection textDirection, {TextStyle textStyle = const TextStyle(fontSize: 400),}) {
```

Comment:
Can you explain the change to this test, I don't fully understand. Thank you!

Reply:
this is to make the text as big as the text widget, so the hittest will hit the textspan for these tests.